website/docs/modules/sources.html.markdown

```diff
@@ -207,6 +207,12 @@ automatically. This is the most common way to access non-public Git
 repositories from automated systems beacuse it is easy to configure
 and allows access to private repositories without interactive prompts.
 
+~> **Note:** The SSH url syntax is different from the SCP syntax (the SCP 
+syntax is what you may be familiar with from GitHub's SSH clone information).
+In Git SSH urls, port is separated from the host by `:` and the directory is 
+separated by `/`, not `:`. If you get an error that Git cannot resolve a
+hostname, this is most likely caused by an incorrect Git SSH url syntax.
```

Comment:
Rather than trying to introduce the difference here by describing surgical changes to the string, perhaps we should instead link to [the official Git documentation](https://git-scm.com/docs/git-clone#_git_urls_a_id_urls_a) and state that "the alternative scp-like syntax" described there is not supported and the `ssh:` URI style must be used instead?

I feel that'll encourage better understanding of what needs to happen, whereas this sentence seems to instead encourage "fiddle with slashes and colons until it works".